app/src/main/java/net/gsantner/markor/ui/hleditor/HighlightingEditor.java

```diff
@@ -45,6 +47,7 @@ public boolean isCurrentLineEmpty() {
     private boolean _hlEnabled = false;
     private boolean _isSpellingRedUnderline;
     private Highlighter _hl;
+    private final Set<TextWatcher> _appliedModifiers = new HashSet<>(); /* Tracks currently applied modifiers */
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5382883)